ReChord_frontend.py

```diff
@@ -110,15 +126,18 @@ def search_terms(path, tag, para):
     """
     results = text_box_search_folder(path, tag, para)
 
-    return render_template('ReChord_result.html', origins=results)
-
+    total_appearance = len(results)
+    if total_appearance != 0:
+        return render_template('ReChord_result.html', origins=results)
+    else:
+        not_found = "No matched term found, maybe try something else?"
+        return render_template('ReChord_result.html', nomatch=not_found)
```

Comment:
Code looks a little nicer if the condition is positive. You also have an extra variable. You can rewrite this as:

```python
if results:
    return render_template('ReChord_result.html', origins=results)
else:
    not_found = "No matched term found, maybe try something else?"
    return render_template('ReChord_result.html', nomatch=not_found)
```

Reply:
deleted the extra variable. What do you mean by positive condition?